torchtitan/parallelisms/parallelize_llama.py

```diff
@@ -329,10 +350,11 @@ def apply_compile(model: nn.Module):
     Apply torch.compile to each TransformerBlock, which makes compilation efficient due to
     repeated structure. Alternatively one can compile the whole model (after applying DP).
     """
+    # avoid recompiling error on 70B model
+    torch._dynamo.config.cache_size_limit = 128
     for layer_id, transformer_block in model.layers.named_children():
-        transformer_block = torch.compile(transformer_block, fullgraph=True)
+        transformer_block = torch.compile(transformer_block)
```

Comment:
We should add fullgraph guard back. Is this for non-llama models? We should comment on which model breaks it from somewhere outside, e.g. in your spreadsheet.

Reply:
ok I'll add this.